skimage/__init__.py

```diff
@@ -56,15 +56,11 @@
 
 """
 
-import os.path as osp
 import imp
 import functools
 import warnings
 import sys
 
-pkg_dir = osp.abspath(osp.dirname(__file__))
```

Comment:
I used `pkg_dir` couple of times to check which installation of `skimage` is currently imported, but `skimage.__file__` works equally well. :+1: